frontend/src/routes/Infrastructure/VirtualMachines/MigrateVirtualMachinePage.tsx

```diff
@@ -0,0 +1,96 @@
+/* Copyright Contributors to the Open Cluster Management project */
+import { useParams, useNavigate } from 'react-router-dom-v5-compat'
+import { AcmModal } from '../../../ui-components'
+import { NavigationPath } from '../../../NavigationPath'
+import { WizardPage } from '../../../wizards/WizardPage'
+import {
+  EditorValidationStatus,
+  Step,
+  useData,
+  useEditorValidationStatus,
+  useItem,
+  WizTextInput,
+} from '@patternfly-labs/react-form-wizard'
+import { t } from 'i18next'
+import { useWizardStrings } from '../../../lib/wizardStrings'
+import { TextContent } from '@patternfly/react-core'
+import { SyncEditor, ValidationStatus } from '../../../components/SyncEditor/SyncEditor'
+import { useTranslation } from '../../../lib/acm-i18next'
+import schema from './schema.json'
+
+export function WizardSyncEditor() {
+  const resources = useItem()
+  const { update } = useData()
+  const { setEditorValidationStatus } = useEditorValidationStatus()
+  const { t } = useTranslation()
+  return (
+    <SyncEditor
+      editorTitle={t('Policy YAML')}
+      variant="toolbar"
+      resources={resources}
+      schema={schema}
+      filters={['*.metadata.managedFields']}
+      immutables={['PlacementBinding.0.*']}
+      onEditorChange={(changes: { resources: any[] }): void => {
+        update(changes?.resources)
+      }}
+      onStatusChange={(editorStatus: ValidationStatus): void => {
+        setEditorValidationStatus(editorStatus as unknown as EditorValidationStatus)
+      }}
+    />
+  )
+}
+
+function getWizardSyncEditor() {
+  return <WizardSyncEditor />
+}
+export default function MigrateVirtualMachinePage() {
```

Comment:
technical suggestion
```suggestion
const getWizardSyncEditor = () => <WizardSyncEditor />

export default function MigrateVirtualMachinePage() {
```